_data/people/sgnoohc.yml

```
@@ -8,6 +8,13 @@ shortname: sgnoohc
 title: Tracking Algorithms Collaborator
 website: https://philip.ucsd.edu
 presentations:
+- title: Line segment tracking for Phase-II Tracking
+  date: 2021-10-18
+  url: https://indico.cern.ch/event/1087315/contributions/4571855/attachments/2329818/3969872/PhilipChang20211018_LST_TrackPOG_v5.pdf
```

Could you like to just the overall contribution, and not the full URL? Indico's full URLs to attachments are not stable.

```suggestion
  url: https://indico.cern.ch/event/1087315/contributions/4571855
```

---

Good point, and I thought about this as well.
But given that there is a `meetingurl` below, which would take you to the meeting indico page, I think this is OK.
(On the website, both links are rendered to be shown next to each other.)

---

It highlights the contribution, at least, and scrolls to it if needed.

---

And it's consistent with what we link to on the hundreds of other presentations.

---

> It highlights the contribution, at least, and scrolls to it if needed.

Ah that's one way to solve it to be more stable and lead to the contribution directly. Yes. I will try that.